source/MapPanel.cpp

```diff
@@ -748,6 +748,8 @@ void MapPanel::Select(const System *system)
 	bool isJumping = flagship->IsEnteringHyperspace();
 	const System *source = isJumping ? flagship->GetTargetSystem() : &playerSystem;
 
+	const System const *previousFinalSystem = plan.empty() ? source : plan.front();
```

Comment:
```suggestion
	const System *previousFinalSystem = plan.empty() ? source : plan.front();
```

Reply:
but whyyyy

Reply:
I want it to not be updated after this line.